shared-bindings/bitmaptools/__init__.h

```diff
@@ -64,6 +64,11 @@ void common_hal_bitmaptools_draw_line(displayio_bitmap_t *destination,
     int16_t x1, int16_t y1,
     uint32_t value);
 
+void common_hal_bitmaptools_draw_circle(displayio_bitmap_t *destination,
+    int16_t x0, int16_t y0,
```

Comment:
Make argnames consistent with public API.
```suggestion
    int16_t x, int16_t y,
```